### ISSUE TITLE
Update to forceLogin

### ISSUE THREAD

**Issue by bobland1** [Schmavery/facebook-chat-api#490]
Simply put as me and @schmavery have found the forceLogin is out of date. I'll look into any possible fixes as accepting login checks automatically is a great benefit

**Comment by Schmavery** [comment]
Just a note that we're not interested in trying to automate any mechanisms facebook is using to attempt to detect whether a human is logging in.  Defeating captchas and the like is an explicit non-goal of the api at this time.

**Comment by bobland1** [comment]
It doesn't involve capachas

**Comment by RandallKent** [comment]
How are folks currently authenticating if they have two-factor enabled? 

I'm getting 

```
{ error: 'Error retrieving userID. This can be caused by a lot of things, including getting blocked by Facebook for logging in from an unknown location. Try logging in with a browser to verify.' }
```

with pulling a new AppState with the code below

```js
var login = require("facebook-chat-api");
var fs = require('fs');
var prompt = require('prompt');
var readline = require("readline");



prompt.start();
prompt.get([{name: 'email', required: true}, { name:'password', hidden: true, required: true}], function (err, result) {
  var email = result.email;
  var password = result.password;
  var appStateFile = '/appstate.json'

  console.log('generating new appstate')

  var rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
  });

  login({email: email, password: password}, {forceLogin: true}, function callback (err, api) {
    if(err) {
      switch (err.error) {
        case 'login-approval':
          console.log('Enter code > ');
          rl.on('line', (line) => {
            err.continue(line);
            rl.close();
          });
          break;
        default:
          console.error(err);
      }
        return;
      }
      fs.writeFileSync(appStateFile, JSON.stringify(api.getAppState()));
  });
});
```

**Comment by Schmavery** [comment]
@RandallKent I don't think this is related to forceLogin.  The 2fa system can be a little flaky so there may be some new issue that needs to be diagnosed.
The original issue tracking this, https://github.com/Schmavery/facebook-chat-api/issues/139, had someone say it was working for them, but that wasn't confirmed by anyone else.  If you want to give a detailed description of your experience with this there, that would be helpful.  Thanks! :)

**Comment by Schmavery** [comment]
I'm going to close this to avoid confusing people.  I will accept any PR that is an extremely simple bypass of a login confirmation page or an update to the docs explaining that force_login will explicitly not handle all cases.
